### DISCUSSION TITLE
2022/03/20/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A81/

### DISCUSSION THREAD

**Discussion by giscus** [OOJ2003/OOJ2003.github.io#20] [Announcements]
# 2022/03/20/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A81/

实现简单链表的第一步

https://ooj2003.github.io/2022/03/20/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A81/

**Comment by OOJ2003** [comment]
www
